apps/src/blockly/themes/cdoJigsaw.js

```diff
@@ -0,0 +1,12 @@
+import * as GoogleBlockly from 'blockly/core';
+
+import {Themes} from '../constants';
+
+import cdoTheme from './cdoTheme';
+
+export default GoogleBlockly.Theme.defineTheme(Themes.JIGSAW, {
+  base: cdoTheme,
+  fontStyle: {
+    size: 24,
```

Comment:
Would a comment on the magic number here and a link to the level that uses it be worthwhile?

Reply:
Yes, good idea. Sadly, there are lots of magic numbers in the existing Jigsaw code too, but no reason to add to that problem. :)

This theme is used in all Jigsaw levels, but I can make that clearer via comment.